examples/ros1/package.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0"?>
 <package format="2">
-  <name>wavemap_examples</name>
+  <name>wavemap_examples_ros1</name>
```

Comment:
Kind of redundant given it's part of ROS1? Or is this needed to not make things cranky once we have ROS2?

Reply:
It was mostly to make it easier to distinguish the example packages (`wavemap_examples_cpp` and later `wavemap_examples_python` and `wavemap_examples_ros2`) and to tightly mirror the structure of the documentation. But we could probably also remove the suffix - it should be fine in terms of ROS1/ROS2 support. I think it'd also be fine to have non-unique CMake project names in the wavemap_ide super project we use for development, although I'd have to try it to be sure.